### PR TITLE
Add multi-task (retrieval + ranking) example

### PR DESCRIPTION
**Pull request by abheesht17**
Delta from TFRS example:

- Data processing, model definition as in the basic ranking/retrieval tutorials (except for a few changes).
- Prediction at the bottom: TFRS shows a rating prediction only. In this example, we have done retrieval first, and then done a rating on the retrieved set.

### CODE REVIEW

File: examples/multi_task.py [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+)
+model.compile(optimizer=tf.keras.optimizers.Adagrad(0.1))
+model.fit(train_ratings, epochs=5)
+
+model.evaluate(test_ratings)
+
+"""
+Let's plot a table of the metrics and pen down our observations:
+
+| Model                 | Top-K Accuracy (↑) | RMSE (↓) |
```
Comment (hertschuh) [inline examples/multi_task.py:340]: Do the little up / down arrows in the table header do anything?

**Review by hertschuh — COMMENTED** [review]
LGTM. Thanks!

Just one question out of curiosity.

File: examples/multi_task.py (hunk 1 above)
Comment (abheesht17) [reply examples/multi_task.py:340]: It just means that a higher value of accuracy is better, and a lower value of RMSE is better

Comment (hertschuh) [reply examples/multi_task.py:340]: Oh ok

**Review by hertschuh — APPROVED** [review]
